cbmc_viewer/coveraget.py

```diff
@@ -227,6 +234,46 @@ def merge_coverage_data(coverage_list):
         raise UserWarning("Error merging coverage data: {}".format(error)) from error
     return coverage
 
+def filter_reachable_functions(coverage, reachable_functions=None):
+    """Restrict coverage data to coverage for reachable functions"""
+
+    if not reachable_functions:
+        return coverage
+
+    def valid_coverage_data(coverage, reachable):
```

Comment:
What does making this an inline function buy us?

Reply:
I don't understand.  It is used on only one place.  What do you recommend that I do?

Reply:
That was just a style question, not a criticism: wrapping it inside a function, vs just inlining the code. I was curious why you'd done the first for code that you only use once.